src/styles/zIndex.js

```diff
@@ -0,0 +1,8 @@
+export default {
+  menu: 10,
+  appBar: 11,
+  leftNav: 12,
+  dialog: 13,
+  layer: 20,
+  popover: 20,
+};
```

Comment:
we definately need tooltip too. add it for now, we can fix #210 and related issues with this